pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -541,44 +545,51 @@ func (b *BridgeBindMechanism) decorateConfig() error {
 	return nil
 }
 
-func (b *BridgeBindMechanism) loadCachedInterface(pid, name string) (bool, error) {
-	ifaceConfig, err := b.storeFactory.CacheForPID(pid).Read(name)
+func getPIDString(pid *int) string {
+	if pid != nil {
+		return fmt.Sprintf("%d", *pid)
+	}
+	return "self"
+}
+
+func (b *BridgeBindMechanism) loadCachedInterface() error {
+	ifaceConfig, err := b.storeFactory.CacheForPID(getPIDString(b.launcherPID)).Read(b.iface.Name)
 	if os.IsNotExist(err) {
-		return false, nil
+		return nil
 	}
 
 	if err != nil {
-		return false, err
+		return err
 	}
 
 	b.virtIface = ifaceConfig
-	return true, nil
+	return nil
 }
 
-func (b *BridgeBindMechanism) setCachedInterface(pid, name string) error {
-	err := b.storeFactory.CacheForPID(pid).Write(name, b.virtIface)
-	return err
+func (b *BridgeBindMechanism) setCachedInterface() error {
+	pid := *b.launcherPID
+	return b.storeFactory.CacheForPID(fmt.Sprintf("%d", pid)).Write(b.iface.Name, b.virtIface)
```

Comment:
Can we use Can we use https://github.com/kubevirt/kubevirt/pull/5059/files#diff-030aa8b7ff9bf4a511d1d5641c1dec04205a944137672be3a691f0b1b4068753R548 ?

Reply:
Same here.